example/00-config.yaml

```diff
@@ -0,0 +1,6 @@
+# SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: garden-shoot-trust-configurator.config.gardener.cloud/v1alpha1
+kind: GardenShootTrustConfiguratorConfiguration
```

Comment:
We can include the different config options (even if commented) in the file so it is easier to understand what could be configured.